components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/js/application.js

```diff
@@ -1,3 +1,39 @@
+$(window).load(function () {
+
+    var client = new SwaggerClient({
+        url: 'https://apis.wso2.com/api/am/store/v0.10/swagger.json',
+        success: function () {
+            client.clientAuthorizations.add("apiKey", new SwaggerClient.ApiKeyAuthorization("Authorization", "Bearer e2994e30-749b-3b54-a294-1eab85abf0f7", "header"));
+            client["Application (Collection)"].get_applications({"responseContentType": 'application/json'},
+                function (data) {
+
+                    $.ajax({
+                        url: '/store/public/components/root/base/templates/appListRow.hbs',
+                        type: 'GET',
+                        success: function (result) {
+                            var theTemplateScript = result;
+                            var theTemplate = Handlebars.compile(theTemplateScript);
+                            var theCompiledHtml, context;
+
+                            for (var i in data.obj.list) {
+

```

Comment:
Fixed in https://github.com/wso2/carbon-apimgt/commit/82f2e62ad13d082ec487bb416a69a72df180d322